File: JavaScript Fundamentals/Homeworks/Conditional Statements/06. Quadratic Equation/quadraticEquation.js
function quadraticEquation() {
    'use strict';

    var a = +document.getElementById('inputFormOne').value,
        b = +document.getElementById('inputFormTwo').value,
        c = +document.getElementById('inputFormThree').value,
        discriminant,
        rootOne,
        rootTwo;

    if(isNaN(a) || isNaN(b) || isNaN(c)) {
        window.console.log('Please enter only numbers !');
        document.getElementById('console').innerHTML += 'Please enter only numbers !' + '<br>';
    }

    else {
        discriminant = (b * b) - (4 * a * c);

        if(discriminant < 0) {
            window.console.log('No real roots !');
            document.getElementById('console').innerHTML += 'No real roots !' + '<br>';
        }

        if(discriminant === 0) {
            rootOne = -(b / (2 * a));
            window.console.log('x1 = x2 = ' + rootOne);
            document.getElementById('console').innerHTML += 'x1 = x2 = ' + rootOne + '<br>';
        }

        if(discriminant > 0) {
            rootOne = (-b + Math.sqrt(discriminant)) / (2 * a);
            rootTwo = (-b - Math.sqrt(discriminant)) / (2 * a);

            window.console.log('x1 = ' + rootOne + ' x2 = ' + rootTwo);
            document.getElementById('console').innerHTML += 'x1 = ' + rootOne + ' x2 = ' + rootTwo + '<br>';
        }
    }
}

File: CSharp Part I/04. Console Input And Output/06. Quadratic Equation/QuadraticEquation.cs
namespace _06.Quadratic_Equation
{
    using System;

    public class QuadraticEquation
    {
        static void Main()
        {
            double a = double.Parse(Console.ReadLine());
            double b = double.Parse(Console.ReadLine());
            double c = double.Parse(Console.ReadLine());
            double descriminant,
                    x1,
                    x2;

            descriminant = (2 * b) - (4 * a * c);

            if(descriminant > 0)
            {
                x1 = (-b + Math.Sqrt(descriminant) / (2 * a));
                x2 = (-b - Math.Sqrt(descriminant) / (2 * a));
                Console.WriteLine(x1);
                Console.WriteLine(x2);
            }
            else if (descriminant == 0)
            {
                x1 = x2 = -b / (2 * a);
                Console.WriteLine(x1);
            }
            else
            {
                Console.WriteLine("no real roots");
            }
        }
    }
}
